### PR TITLE
Fix silent failures in unit tests since global object refactoring

### PR DESCRIPTION
**Pull request by baxtersa**

### CODE REVIEW

**Comment by baxtersa** [comment]
@arjunguha , assuming the tests pass, you probably want to take a look at this before I merge.

**Comment by baxtersa** [comment]
Just wanted to ping this again. This also includes a fix for #403

File: stopify-continuations/src/runtime/abstractRuntime.ts [hunk 1]
```diff
@@ -152,8 +152,8 @@ export abstract class Runtime implements RuntimeInterface {
           this.mode = false;
           this.stack = result.stack;
           this.savedStack = result.savedStack;
-          const params = (<KFrameRest>this.stack[this.stack.length - 1]).params || [];
-          return this.stack[this.stack.length - 1].f.apply(global, params);
+          const frame = <KFrameRest>this.stack[this.stack.length - 1];
+          return this.stack[this.stack.length - 1].f.apply(frame.this || global, frame.params || []);
```
[the remark this answers is not in the document]
Comment (baxtersa) [reply stopify-continuations/src/runtime/abstractRuntime.ts]: Ah, good point. I think I was over-complicating things. I'll work on this now.

**Comment by arjunguha** [comment]
LGTM

**Pull request merged**